src/main-process/window.js

```diff
@@ -24,23 +23,44 @@ fluid.defaults("electron.browserWindow", {
     showOnCreate: false,
     showOnReady: true,
 
-    windowOptions: {},
+    windowOptions: {
+        webPreferences: {
+            nodeIntegration: false
+        },
+
+        width: 100,
+        height: 100,
+        x: 0,
+        y: 0
+    },
 
     members: {
-        win: "@expand:electron.browserWindow.create({that}.options.windowOptions)"
+        win: {
+            expander: {
+                funcName: "electron.browserWindow.create",
+                args: ["{that}.options.windowOptions"]
+            }
+        }
+    },
+
+    invokers: {
+        show: {
```

Comment:
This can probably be axed, esp. since "hide" was never implemented. If we have modelised some part of application function, this serves as a preferable interface to "invokers" which will be abolished in any case